docs/best_practices/image_series.rst

```diff
@@ -0,0 +1,25 @@
+ImageSeries
+===========
+
+
+
+Storage of ImageSeries
+----------------------
+
+Use external model when file is too large
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+When storing commmon multimedia formats using the :nwb-schema:ref:`sec-ImageSeries` the file should be stored as
+an external file. That is, the file should be packaged together with the nwb file instead of stored inside the format.
+This can be accomplished by using  the ``external_file`` file option to store the path instead.
+
+Check function: :py:meth:`~nwbinspector.checks.check_image_series_too_large`
+
+
+Use relative path for external mode
```

Comment:
A really cool thing with our custom sphinx code here is you can form an API doc linkage with the BP section by defining a reference to this section of the following form: `.. _best_practice_<name_of_check_function_that_tests_it>:` or specific to this section `.. _best_practice_image_series_external_file_relative:`

Can you add these links here for both sections?

Reply:
OK, see the ones that I add to see if I got you.